fdbserver/storageserver.actor.cpp

```diff
@@ -10962,7 +10980,7 @@ ACTOR Future<Void> watchValueWaitForVersion(StorageServer* self,
 	getCurrentLineage()->modify(&TransactionLineage::txID) = req.spanContext.traceID;
 	try {
 		wait(success(waitForVersionNoTooOld(self, req.version)));
-		self->checkTenantEntry(latestVersion, req.tenantInfo);
+		self->checkTenantEntry(latestVersion, req.tenantInfo, false);
```

Comment:
I started the implementation, but it is quite cumbersome. I am not sure whether we need this, so I left it out for now. Should we need it I can add it in a separate PR (it will require quite a few changes across some change-hot files)